docs/External_HSS_Integration.md

```diff
@@ -71,6 +71,28 @@ In order to register and make calls, you need to create subscriber records on yo
 2. create an associated private user identity with its ID formed by removing the `sip:` scheme prefix from the public user ID
 3. configure the public user identity's Initial Filter Criteria to include an application server named `sip:mmtel.your.home.domain`, where `your.home.domain` is replaced with your home domain - this enables MMTEL services for this subscriber.
 
+### Allowing one subscriber to have two private identities
+
+If you wish to use a stock Android SIP client to make calls through Clearwater (for details
```

Comment:
Slightly ambiguous language (is it Clearwater that doesn't contain an Authentication username field? does the client default to this username if I don't wish to make calls?). Try:

If you try to use an [Android SIP client](Configuring_the_native_Android_SIP_client.md#instructions-1)  that doesn't contain an **Authentication username** field, the client will default to a username...